### DISCUSSION TITLE
Spinner animation is slower than on stable

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#24306] [Q&A] [answered]
Noticed this when using a skin that doesn't use a perfect circle as the spinner-circle sprite.

On stable one rotation on a spinner will also make the spinner-circle rotate once, while on lazer you need two rotations for the spinner-circle to make one full rotation. This can be a bit irritating as it can give you the illusion that you're spinning slower than you actually are. In the videos below you can see that I'm spinner faster on lazer, however visually it looks like I'm spinning a lot slower.

https://github.com/ppy/osu/assets/22381687/573ce7c5-0f8b-4c7f-ade0-89ae3ee394b2

https://github.com/ppy/osu/assets/22381687/68c92220-2ef9-426e-8c06-c08eb388ae91

beatmap: https://osu.ppy.sh/beatmapsets/36323#osu/117379
skin: https://drive.google.com/file/d/1I1Z_oiHaf3Pj-Pjeacv641R21Ak5yNGe/view?usp=sharing

**Comment by integerrr** [comment]
Not against "fixing" it but do you not just look at the rpm count to determine how fast you're spinning? Personally it never bothered me with a wonky spinner skin

**Reply by respektive** [reply]
I do, but dunno I'm just kinda used to the spinner going brrrr like on stable I guess, so it looks kinda off.
It's not really that important in the end, just another small difference between lazer and stable i thought is worth bringing up. :man_shrugging:

**Reply by integerrr** [reply]
Fair enough 👍

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/24360
